insights/specs/sos_archive.py

```diff
@@ -88,6 +88,9 @@ class SosSpecs(Specs):
     pcs_config = simple_file("sos_commands/pacemaker/pcs_config")
     pcs_quorum_status = simple_file("sos_commands/pacemaker/pcs_quorum_status")
     pcs_status = simple_file("sos_commands/pacemaker/pcs_status")
+    podman_list_containers = first_file(["sos_commands/podman/podman_ps_-a", "sos_commands/podman/podman_ps"])
```

Comment:
You may forget to add the `podman_container_inspect`

Reply:
Nope, podman_container_inspect refers to the same file locations i.e. podman_inspect_*

Kept in uniform to what has been implemented in docker:
https://github.com/RedHatInsights/insights-core/blob/master/insights/specs/sos_archive.py#L34